__tests__/app.test.js

```diff
@@ -98,22 +145,50 @@ describe("GET/api/reviews/:review_id", () => {
               "You have stumbled across an uncharted island rich in natural resources, but you are not alone; other adventurers have come ashore too, and the race to settle the island of Catan has begun! Whether you exert military force, build a road to rival the Great Wall, trade goods with ships from the outside world, or some combination of all three, the aim is the same: to dominate the island. Will you prevail? Proceed strategically, trade wisely, and may the odds be in favour.",
             category: "social deduction",
             created_at: expect.any(String),
-            votes: 16,
+            votes: 26,
           })
-        );
+         );
       });
   });
-  test("400: return 'invalid input' when given an invalid id type", () => {
+  test("400: returns 'Invalid Data' when given inc_votes of the wrong type", () => {
+    const requestObject = { inc_votes: "null" };
+
     return request(app)
-      .get("/api/reviews/string")
+      .patch("/api/reviews/13")
+      .send(requestObject)
       .expect(400)
       .then((response) => {
         expect(response.body).toEqual({ msg: "Invalid Input" });
       });
   });
-  test("404: returns 'Not Found' when given an ID number not contained in the db", () => {
+  test("400: returns 'Invalid' when given review_id of incorrect type", () => {
+    const requestObject = { inc_votes: 10 };
+
     return request(app)
-      .get("/api/reviews/999")
+      .patch("/api/reviews/random")
+      .send(requestObject)
+      .expect(400)
+      .then((response) => {
+        expect(response.body).toEqual({ msg: "Invalid Input" });
+      });
+  });
+  test("400: returns 'Invalid' when given broken requestObject", () => {
```

Comment:
This test is fine as it is, but keep an open mind about using future expansion of the database where we could patch with other similar objects for example {inc_rating: 1}. 

For now no change needed.

Reply:
Okay, I'll bear that in mind thanks!